README.md

```diff
@@ -25,25 +24,24 @@ Or, of course, install it in any other way you want.
 
 ### Pin versions
 
-When referencing third-party actions, [always pin the version to a specific
-commit hash][hardening]. This ensures that the workflow will always use the same
-version of the action, even if the action's maintainers release a new version or
-if the tag itself is updated.
+When using third-party actions, [always pin the version to a specific commit hash][hardening].
+This ensures that the workflow always uses the same version of the action, even if the action's maintainers release a new version or update the Git tag.
 
-Dependabot can update these SHA references when there are new versions. If you
-include a tag in a commend after the SHA, it can update the comment too. For
-example:
+Dependabot updates these SHA references when there are new versions.
+If you include a tag in a commend after the SHA, it updates the comment too.
+For example:
 
 ```yaml
 - uses: action/foo@abcdef0123456789abcdef0123456789 # v1.2.3
 ```
 
 [hardening]: https://docs.github.com/en/actions/security-guides/security-hardening-for-github-actions#using-third-party-actions
 
-### Refer to other `shared-workflows` actions using relative paths
+### Use other `shared-workflows` actions with relative paths
```

Comment:
Just picking an instance at random here. I like the editing you're doing, but I'd like it more if we could enforce it so that you don't need to pop up every now and again and fix our style. Is there a way we can do this? Or at the least, is there a style guide we can refer to?

As I was writing that paragraph I realised that this pull request contains two changes: adding `shellcheck` and copyediting.

I won't _require_ it but it would be better to split this into two. And then in the other one we can discuss how to keep ourselves on the straight and narrow.

Reply:
Opened https://github.com/grafana/shared-workflows/pull/475